docs/plugins/contributors/gatsby-node.js

```diff
@@ -93,22 +116,10 @@ exports.sourceNodes = async ({ actions, createContentDigest, createNodeId }, { r
       });
     }
   } catch (error) {
-    staticData.forEach(user => {
-      createNode({
-        ...user,
-        error: warnMissingAccessToken(error),
-        avatar_url: "",
-        parent: null,
-        children: [],
-        internal: {
-          type: NODE,
-          content: JSON.stringify(user),
-          contentDigest: createContentDigest(user),
-        },
-      });
-    });
     console.error(error);
-    warnMissingAccessToken(error);
+    console.info(
+      "You may have forgotten to include the repo and read:org scopes for your GitHub access token.",
+    );
```

Comment:
Clumsily adapting this logic to `dotenv-safe` instead of `dotenv`. We'll come up with better dotenv utilities later.